packages/hdwallet-native/src/ethereum.ts

```diff
@@ -52,6 +52,7 @@ export function MixinNativeETHWallet<TBase extends core.Constructor<NativeHDWall
   // eslint-disable-next-line @typescript-eslint/no-shadow
   return class MixinNativeETHWallet extends Base {
     readonly _supportsETH = true;
+    readonly _supportsEthSwitchChain = true;
```

Comment:
remove for now until this is true

Reply:
Removed in b3852ba8a44e80c11c1a5f2804dbcf687d3b4116